docs/_ext/autorobusta.py

```diff
@@ -244,37 +225,29 @@ def run(self) -> List[Node]:
         action_definition = Action(obj)
         return self.__generate_rst(action_definition, recommended_trigger)
 
-    def __generate_rst(
-        self, action_definition: Action, recommended_trigger: Optional[str]
-    ):
+    def __generate_rst(self, action_definition: Action, recommended_trigger: Optional[str]):
         node = nodes.section()
         node.document = self.state.document
 
         trigger_params = json.loads(self.options.get("trigger-params", "{}"))
 
-        example_yaml = generator.generate_example_config(
-            action_definition.func, recommended_trigger, trigger_params
-        )
+        example_yaml = generator.generate_example_config(action_definition.func, recommended_trigger, trigger_params)
         params_cls = action_definition.params_type
         params_cls_path = ""
         if params_cls is not None:
             params_cls_path = f"{params_cls.__module__}.{params_cls.__name__}"
 
-        code = self.__get_source_code(action_definition.func)
+        # code = self.__get_source_code(action_definition.func)
```

Comment:
why was this commented?

Reply:
Because it was actually not used anywhere

Reply:
Its cool that it automatically comments out unused code, 
lets find all the unused code and remove it

Reply:
No, flake8 does not do it automatically, just warns you that it is unused, so I commented it out.
Maybe we should not remove it, because it might be usefull in the future?